src/roiextractors/extractors/nwbextractors/nwbextractors.py

```diff
@@ -336,10 +336,14 @@ def get_images_dict(self):
 
         return images_dict
 
-    @property
-    def roi_locations(self):
-        if self._roi_locs is not None:
-            return self._roi_locs.data[:].T
```

Comment:
This method was never working properly - it was always leading to the base method anyway